src/ProcessMaker/Nayra/Bpmn/IntermediateCatchEventTrait.php

```diff
@@ -120,5 +121,10 @@ public function execute(EventDefinitionInterface $message, ExecutionInstanceInte
         $this->triggerPlace->addNewToken($instance);
         return $this;
     }
+
```

Comment:
Add a register method for the catch event elements like intermediate catch events.

Reply:
Completed